### DISCUSSION TITLE
Support for WPML

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-future#350] [Feature Requests]
Moving from this post:
https://wordpress.org/support/topic/expiring-other-languages-posts-wpml/

Here’s our method when creating a post/page and setting the expiration (and it’s language “siblings”):

1- Page created on default language, english (New offer):
/wp-admin/post.php?post=27339&action=edit&lang=en&classic-editor

2- Duplication to other languages:

![index](https://user-images.githubusercontent.com/3868028/159793092-c1ffbd57-4add-4e10-9c63-6bcbee2a4b21.png)

3-Setting up a specific date and time to the default language page to expire (18h)

![index](https://user-images.githubusercontent.com/3868028/159793164-58c60c34-8042-4922-9575-c011f03a9be9.png)

4- Opening every other language pages, confirming the same date and time are applied and Updating post.

![index](https://user-images.githubusercontent.com/3868028/159793236-07725689-1d78-4d61-99ae-3fba540eb74c.png)

5- Events created by the plugin (log): 

![index](https://user-images.githubusercontent.com/3868028/159793293-3b4cd5d2-8f89-4966-8969-e38149a4e16a.png)

6- At 18h, the 5 emails were sent warning about the status change:

![index](https://user-images.githubusercontent.com/3868028/159793408-b5097cbd-bdf8-428a-9d08-62648c5953b1.png)

The problem here is that only the English page (default language) expired to draft. The other 4, didn’t expired.

**Comment by stevejburge** [comment]
One more request for this

**Comment by rizaardiyanto1412** [comment]
One more request: https://wordpress.org/support/topic/wpml-compatible-85/

**Comment by ZoltanGalCMDesign** [comment]
I also really need this feature. Thanks.

**Comment by kevine2124** [comment]
I also really need this, is there any update for this topic?

**Comment by rizaardiyanto1412** [comment]
@keenti This is on our feature request box. Meaning the possibilities this coming in the future release is depend on the vote. We work based on priorities. The more upvotes the idea receive, the more likely it will come into features. 

You can upvote it here

![image](https://user-images.githubusercontent.com/83811723/226610663-532c2a8e-1035-416a-b200-585da5b03c63.png)

**Comment by danielvigh** [comment]
I also really need this feature. Thanks.
